### PR TITLE
Spellcheck Updates For Concept and Guides Pages

### PR DESCRIPTION
**Pull request by chrisVillanueva**
small spelling and syntax changes.

### CODE REVIEW

File: content/guides/lazy-load-react.md [hunk 1]
```diff
@@ -3,13 +3,14 @@ title: Lazy Loading - React
 sort: 14
 contributors:
   - iammerrick
+  - chrisVillanueva
 ---
 
-A component can lazily load dependencies without its consumer knowing using higher order functions, or a consumer can lazily load its children without its children knowing using a component that takes a function and collection of modules, or some combination of both.
+A component can lazily load dependencies without its consumer knowing using higher order functions, or a consumer can lazily load it's children without its children knowing using a component that takes a function and collection of modules, or some combination of both.
```
Comment (SpaceK33z) [inline content/guides/lazy-load-react.md]: Shouldn't it be `its children`?

Comment (bebraw) [inline content/guides/lazy-load-react.md]: Yup.

**Comment by bebraw** [comment]
Thanks. 👍

**Comment by chrisVillanueva** [comment]
No problem @bebraw .  I'm happy to contribute as much as possible in my spare time.
Sorry about the initial mistake, as I was in a small time crunch.  

I made the correction and will be more thorough with future PRs.


Chris

**Pull request merged**